src/program/snabbnfv/doc/getting-started.md

```diff
@@ -0,0 +1,610 @@
+             471,424 sent on nicB.tx -> sink.in2 (loss rate: 0%)
+             939,675 sent on source1.out -> nicA.rx (loss rate: 0%)
+             939,675 sent on source2.out -> nicB.rx (loss rate: 0%)
+-------
+Send traffic from a nicA (SF) to nicB (two VFs)
+The packets should arrive evenly split between the VFs
+link report:
+                   0 sent on nicAs.tx -> sink_ms.in1 (loss rate: 0%)
+             560,558 sent on nicBm0.tx -> sink_ms.in2 (loss rate: 0%)
+             560,665 sent on nicBm1.tx -> sink_ms.in3 (loss rate: 0%)
+           1,121,745 sent on repeater_ms.output -> nicAs.rx (loss rate: 0%)
+                   2 sent on source_ms.out -> repeater_ms.input (loss rate: 0%)
+selftest: ok
+```
+
+## Create and launch two VM's
+
+Now that snabbswitch can talk to both 10GbE ports successfully, lets build and launch 2 test VM's and connect each of them to one of the 10GbE port. First, we have to build an empty disk, download and install Ubuntu in it:
+
+Create a disk for the VM:
+
+```
+$ qemu-img create -f qcow2 ubuntu.qcow2 16G
+```
+	
+Download ubuntu server 14.04.2:
+
+```
+$ wget http://releases.ubuntu.com/14.04.2/ubuntu-14.04.2-server-amd64.iso
+```
+	
+Launch the ubuntu installer via qemu and connect to its VNC console running at <host>:5901. This can be done via a suitable VNC client.
+
+```
+$ sudo qemu-system-x86_64 -m 1024 -enable-kvm \
+-drive if=virtio,file=ubuntu.qcow2,cache=none \
+-cdrom ubuntu-14.04.2-server-amd64.iso -vnc :1
+```
+
+The installer guides you thru the setup of ubuntu. I picked username ubuntu with password ubuntu and use the whole disk without LVM, no automatic updates and selected openssh as the only optional package to install.
+Kill qemu after the reboot. 
+
+We have now a master VM ubuntu virtual disk to create two VM's from and launch them individually. Create first two copies:
+
+```
+$ cp ubuntu.qcow2 ubuntu1.qcow2
+$ cp ubuntu.qcow2 ubuntu2.qcow2
+```
+	
+Before launching the VM's, we need to prepare snabb to work as virtio interface for the VM's. Snabb offers snabnfv traffic app for this, which is built-into the snabb binary that was built earlier. Source and documentation can be found at [https://github.com/SnabbCo/snabbswitch/tree/next/src/program/snabbnfv](https://github.com/SnabbCo/snabbswitch/tree/next/src/program/snabbnfv)
+
+One Snabbnfv traffic process is required per 10 Gigabit port and uses a configuration file with port information for every vhost interface:
+
+* VLAN
+* MAC address of the VM
+* Id, which is used to identify a socket name
+
+VLAN and MAC are used to pass ethernet frames based on destination address to the correct vhost interface. I created a separate config file per 10GbE port.
+
+```
+$ cat port1.cfg
+return {
+  { vlan = 431,
+    mac_address = "52:54:00:00:00:01",
+    port_id = "id1",
+  },
+}
+$ cat port2.cfg
+return {
+  { vlan = 431,
+    mac_address = "52:54:00:00:00:02",
+    port_id = "id2",
+  },
+}
+```
```

Comment:
Might be worthwhile to point out that every Snabb NFV configuration can define multiple ports so that `n` VMs can connect to one 10G port.